docs/en/ingest-management/elastic-agent/configuration/inputs/simplified-input-configuration.asciidoc

```diff
@@ -0,0 +1,25 @@
+[[elastic-agent-simplified-input-configuration]]
+= Configure simplified log ingestion for Standalone {agent}s
+
+include::{fleet-repo-dir}/standalone-note.asciidoc[]
+
+
+There is a simplified option for ingesting log files with the
+Elastic-Agent, the simplest input configuration to ingest the file
+`/var/log/my-application/log-file.log` is:
+
+["source","yaml"]
+-----------------------------------------------------------------------
+inputs:
+  - type: filestream <1>
+    id: unique-id-per-input <2>
+    paths: <3>
+      - /var/log/my-application/log-file.log
+-----------------------------------------------------------------------
+
+<1> The input type must be `filestream`.
+<2> A unique ID for the input.
+<3> An array containing all log file paths.
+
+For other custom options to configure the input, refer to the
+<<filebeat-input-filestream,filestream input documentation>>.
```

Comment:
```suggestion
For other custom options to configure the input, refer to the {filebeat} {filebeat-ref}/filebeat-input-filestream.html[filestream input] in the {filebeat} documentation.
```
@belimawr I know this is still in draft so I hope you don't mind my jumping in. The suggestion above would create a link to [here](https://www.elastic.co/guide/en/beats/filebeat/master/filebeat-input-filestream.html) and should also fix up that CI check failure, just in case that helps.

Reply:
Thanks a lot @kilfoyle! I added it to my PR.